src/main/java/uk/gov/hmcts/reform/notificationservice/data/NotificationRepository.java

```diff
@@ -72,7 +75,8 @@ public NotificationRepository(
     public List<Notification> findPending() {
         return jdbcTemplate.query(
             "SELECT * FROM notifications WHERE status = :status and confirmation_id IS NULL and "
-                + "created_at < (now()::timestamp - interval '" + delayDurationToProcessPending + " minutes')",
+                + "created_at < (now()::timestamp - interval '" + delayDurationToProcessPending + " minutes') "
+                + "ORDER BY id",
```

Comment:
We definitely do not need this change - `findPending` method is called from `PendingNotificationsTask`, not from the controller.